app/models/katello/host/content_facet.rb

```diff
@@ -18,6 +18,8 @@ class ContentFacet < Katello::Model
       validates :host, :presence => true, :allow_blank => false
       validates_with Validators::ContentViewEnvironmentValidator
 
+      attr_accessible :content_view, :content_view_id, :lifecycle_environment, :lifecycle_environment_id, :host
```

Comment:
Three options:

1. Set the values explicitly on the model (e.g. https://gist.github.com/daviddavis/c89bf440cc2d763ccc43)
2. Use a ActionController::Parameters object and permit the keys you want (see http://api.rubyonrails.org/v4.2.4/classes/ActionController/Parameters.html#method-i-permit)
3. Use a factory